examples/animation/bezier_curve.rs

```diff
@@ -0,0 +1,99 @@
+//! Demonstrates how to work with beizer curves.
+//!
+
+use bevy::{math::cubic_splines::CubicCurve, prelude::*};
+
+#[derive(Component)]
+pub struct CubeCurve(CubicCurve<Vec3>);
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .add_systems(Startup, setup)
+        .add_systems(Update, animate_cube)
+        .run();
+}
+
+fn setup(
+    mut commands: Commands,
+    mut meshes: ResMut<Assets<Mesh>>,
+    mut materials: ResMut<Assets<StandardMaterial>>,
+) {
+    // Define your control points
+    // These point will build a curve looking like
+    //
+    // y
+    // │     .``.
+    // │    .-``-.
+    // │ .-`      `-.
+    // ●─────────────── x
+    //
+    // (Do not comment my ascii art graph skills)
```

Comment:
I don't think this text is needed. You can see the curve when running the example, and it doesn't really explain control points. Maybe link to a wikipedia article?

Reply:
That's more relevant indeed. Changed in commit adefa61db1b45013e8c278ffb4aea754404cace2